backend/main.py

```
@@ -82,17 +100,20 @@ def ingest_sentry(payload: dict):
             title=title,
             body=body,
             metadata=payload,
-            created_at=datetime.now(timezone.utc)
+            created_at=datetime.now(timezone.utc),
         )
         add_feedback_item(item)
         return {"status": "ok", "id": str(item.id)}
 
     except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)